recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "1.5.12" %}
+{% set version = "1.5.13" %}
 package:
   name: libmatio
   version: {{ version }}
 
 source:
   fn: matio-{{ version }}.tar.gz
   url: https://github.com/tbeu/matio/releases/download/v{{ version }}/matio-{{ version }}.tar.gz
-  md5: 2968b5a95f05fab8b79ec87f9a6c0cac
+  md5: 86b3294c64a4fe8be2a8f6a7840f5bf3
   patches:
     - 0001-hotfix-backport-from-92.patch  # [linux]
```

Comment:
Looks like this patch no longer applies cleanly.

ref: https://circleci.com/gh/conda-forge/libmatio-feedstock/59